x-pack/plugins/apm/scripts/optimize_tsconfig/tsconfig.json

```diff
@@ -1,8 +1,9 @@
 {
   "include": [
+    // "./x-pack/plugins/apm/server/routes/services/route.ts",
     "./x-pack/plugins/apm/**/*",
-    "./x-pack/plugins/observability/**/*",
-    "./x-pack/plugins/rule_registry/**/*",
+    // "./x-pack/plugins/observability/**/*",
```

Comment:
if we don't need these entries, can we just remove them?

Reply:
ahh thanks, I forgot to!